src/message.ts

```diff
@@ -1,8 +1,15 @@
 import { Time } from './common'
 
 export interface IBlueprintExternalMessageQueueObj {
+	/** If set, the message references an existing message (that is to be overrritten) */
+	_id?: string
+
 	/** Type of message */
 	type: IBlueprintExternalMessageQueueType
+	/** If true, the message won't be sent automatically */
+	queueForLater?: boolean
```

Comment:
Do we need both queueForLater and queueForLaterReason? Could we not have just queueForLater, and we check it for truthiness/being defined instead?

Reply:
Good idea 🙂